cpp/open3d/geometry/TriangleMesh.h

```diff
@@ -361,28 +361,33 @@ class TriangleMesh : public MeshBase {
             bool use_triangle_normal,
             int seed);
 
-    /// Function to sample \param number_of_points points uniformly from the
-    /// mesh. \param use_triangle_normal Set to true to assign the triangle
-    /// normals to the returned points instead of the interpolated vertex
-    /// normals. The triangle normals will be computed and added to the mesh
-    /// if necessary. \param seed Sets the seed value used in the random
-    /// generator, set to -1 to use a random seed value with each function call.
+    /// Function to sample points uniformly from the mesh.
```

Comment:
functions in `TriangleMesh.h` seems to be missing `\brief` keyword. Is it required?